e2e/app/eoa/testdata/threshold_reference.json

```diff
@@ -14,8 +14,8 @@
     "target": "0.0100"
    },
    "hot": {
-    "min": "2.0080",
-    "target": "8.0800"
+    "min": "22.0080",
+    "target": "108.0800"
```

Comment:
these thresholds are too high for mainnet

Reply:
I think threshold for tester has to be decreased on mainnet, possibly we can do a dynamic overwrite of the threshold, see something similar here: https://github.com/omni-network/omni/blob/16feed3cd72b61f0b915ffe2fc5095edf35097ab/e2e/app/eoa/fund.go#L74